cli/cmd/asset_create.go

```diff
@@ -0,0 +1,148 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openclarity/vmclarity/api/models"
+	cliutils "github.com/openclarity/vmclarity/cli/pkg/utils"
+	"github.com/openclarity/vmclarity/pkg/shared/backendclient"
+)
+
+// assetCreateCmd represents the standalone command.
+var assetCreateCmd = &cobra.Command{
+	Use:   "asset-create",
+	Short: "Create asset",
+	Long:  `It creates asset. It's useful in the CI/CD mode without VMClarity orchestration`,
+	Run: func(cmd *cobra.Command, args []string) {
+		logger.Infof("Creating asset...")
+		filename, err := cmd.Flags().GetString("from-json-file")
```

Comment:
Now we don't need to maintain all the asset type in the cli because now we can use `assetType.Unmarshal`  that generated from swagger. In the case of the yaml input for unmarshal we need to define the asset type struct in cli. The other option is converting yaml input into json.

Reply:
Decided to support only json first because of the generated API code (that doesn't support YAML straightforward). We'll handle YAML in a separate task